libraries/recipe_utils.rb

```diff
@@ -139,13 +139,14 @@ def self.use_splunk_start_command?(node, previous_splunk_version)
   end
 
   # Returns the Splunk service name based on platform and package name
-  def self.splunk_service_name(platform_family, package_base_name)
+  def self.splunk_service_name(platform_family, package_base_name, systemd_unit_file_name)
     if platform_family == 'windows'
       return 'SplunkForwarder' if package_base_name == 'splunkforwarder'
       return 'Splunkd' if package_base_name == 'splunk'
     end
-
-    'splunk'
+      return systemd_unit_file_name
+    #return 'splunkforwarder' if package_base_name == 'splunkforwarder'
+    #'splunk'
```

Comment:
fix indentation and remove commented code.

Reply:
Removed: https://github.com/cerner/cerner_splunk/pull/265/commits/8f3f110b8b4b23558dafe16fa9d191478f576e58